qiskit_experiments/framework/base_experiment.py

```diff
@@ -39,14 +39,21 @@ def __init__(
         analysis: Optional[BaseAnalysis] = None,
         backend: Optional[Backend] = None,
         experiment_type: Optional[str] = None,
+        **experiment_options,
     ):
         """Initialize the experiment object.
 
+        `experiment_options` catches all keyword arguments not explicitly defined in `__init__`. As this
+        is the base class, `experiment_options` should be an empty dictionary. If it is not empty, then a
```

Comment:
This sentence doesn't really make sense to me and contradicts how it is used in the other modified experiments. I think this whole paragraph should be removed.

Reply:
I've removed this paragraph in bbe6821.